[spaceship] update error message tooManyCodesSent 2FA and 2SA

bolom opened the pull request:
<!-- Thanks for contributing to _fastlane_! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->

### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

tooManyCodesSent error message was not raised for 2FA and 2SA

max-ott committed:
Merge pull request #1 from fastlane/master

Update

max-ott committed:
Merge pull request #2 from fastlane/master

Upgrade

bolom committed:
update error message tooManyCodesSent 2FA and 2SA

joshdholtz commented on spaceship/lib/spaceship/two_step_or_factor_client.rb:
```diff
@@ -25,7 +25,7 @@ def handle_two_step_or_factor(response)
     end
 
     def handle_two_step(response)
-      if response.body.fetch("securityCode", {})["tooManyCodesLock"].to_s.length > 0
```
Is `tooManyCodesLock` not a thing anymore? 🤔 Should we maybe keep checking for it just in case?

joshdholtz replied:
@bolom Just wanted to check in to see your thoughts on this? 😊

bolom replied:
should we use securityCodeLocked for 4 digit and tooManyCodesSent for 6 digits?

joshdholtz reviewed (commented):
This looks good! Just one though on keeping `tooManyCodesLock` still just as backup 🤷‍♂️

Commit:
Add more endpoints

max-ott committed:
Merge pull request #3 from fastlane/master

Bump version

Commit:
Add delete screenshots and preview sets

max-ott committed:
Merge pull request #1 from max-ott/master

Bump

Commit:
Add additional API methods

Commit:
Add update referrer

max-ott committed:
Merge pull request #2 from fastlane/master

[spaceship] Add support for deleting app_screenshot_sets and app_prev…

Commit:
Add more AppAnalytics options

Commit:
Gemfile

max-ott committed:
Merge branch 'mo-analytics' into master

max-ott committed:
Merge pull request #5 from max-ott/master

Get master

Commit:
Add dimensions array at the end

Commit:
Merge branch 'mo-analytics' of https://github.com/max-ott/fastlane into mo-analytics

max-ott committed:
Merge pull request #3 from max-ott/mo-analytics

Mo analytics

joshdholtz commented:
@bolom Hello 👋  Are you still looking to get this merged? 😊 It has some commit from another branch/PR so I think those need to get cleaned up before this can get merged 🤷‍♂️

Commit:
fix conflict

bolom commented:
@joshdholtz yes please

max-ott committed:
Fix case sensitive email issue

Apple treats emails case sensitive. Sample: `Spaceship::Members.all` => `"emil.dvorjak@acme.com", "Stacy.Hammilton@acme.com", "samual.Ethan@acme.com"` 

If you search for `Spaceship::Members.find("stacy.hammilton@acme.com")`, the call will fail, because there is no match. This PR tries to fix the issue.

max-ott committed:
Merge pull request #4 from fastlane/max-ott-patch-3

Max ott patch 3

Commit:
fix conflict

Commit:
add casecmp

Commit:
update gem

Commit:
update gem

Commit:
update gem

Commit:
merge master

Kukurijek commented:
Hi @bolom!

Could you rebase the master and resolve the conflicts please 😄 ?

rogerluan commented:
Since the author deleted the branch where this code was hosted, I'm assuming there's no interest to get this PR merged anymore, so I'm closing it 🙇 

Should this change, feel free to comment here and I can reopen this PR (assuming the branch is up-to-date and in a mergeable state) 🙏

bolom commented:
@bolom yes you can close it
